src/styles/states.css

```diff
@@ -55,6 +55,6 @@
 }
 
 .state--unknown { @apply
-  bg-background-300
-  text-foreground-700
+  text-foreground-700;
+  background-color: var(--p-color-sentiment-neutral);
```

Comment:
isn't there a tailwind utility for sentiment neutral?

Reply:
There totally is, will update